exchanges/bybit/bybit_test.go

```diff
@@ -85,3521 +80,3401 @@ func TestGetKlines(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-}
-
-func TestGetLCPInfo(t *testing.T) {
-	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	pair, err := currency.NewPairFromString("BTCUSD")
+	// USDT Perp open long position (one-way mode)
+	arg = &PlaceOrderParams{Category: "linear",
+		Symbol: usdcMarginedTradablePair, Side: "Buy", OrderType: "Limit", OrderQuantity: 1, Price: 25000, TimeInForce: "GTC", PositionIdx: 0, OrderLinkID: "usdt-test-01", ReduceOnly: false, TakeProfitPrice: 28000, StopLossPrice: 20000, TpslMode: "Partial", TpOrderType: "Limit", SlOrderType: "Limit", TpLimitPrice: 27500, SlLimitPrice: 20500}
+	_, err = b.PlaceOrder(context.Background(), arg)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-
-	_, err = b.GetLiquidityContributionPointsInfo(context.Background(), pair)
+	// USDT Perp close long position (one-way mode)
+	arg = &PlaceOrderParams{Category: "linear", Symbol: usdtMarginedTradablePair, Side: "Sell",
+		OrderType: "Limit", OrderQuantity: 1, Price: 3000, TimeInForce: "GTC", PositionIdx: 0, OrderLinkID: "usdt-test-02", ReduceOnly: true}
+	_, err = b.PlaceOrder(context.Background(), arg)
 	if err != nil {
 		t.Error(err)
 	}
 }
 
-func TestGetFutureWalletBalance(t *testing.T) {
+func TestAmendOrder(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetFutureWalletBalance(context.Background(), "BTC")
+	if mockTests {
+		t.Skip(skipAuthenticatedFunctionsForMockTesting)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
+	_, err := b.AmendOrder(context.Background(), nil)
+	if !errors.Is(err, errNilArgument) {
+		t.Fatalf("expected %v, got %v", errNilArgument, err)
+	}
+	_, err = b.AmendOrder(context.Background(), &AmendOrderParams{})
+	if !errors.Is(err, errEitherOrderIDOROrderLinkIDRequired) {
+		t.Fatalf("expected %v, got %v", errEitherOrderIDOROrderLinkIDRequired, err)
+	}
+	_, err = b.AmendOrder(context.Background(), &AmendOrderParams{
+		OrderID: "c6f055d9-7f21-4079-913d-e6523a9cfffa",
+	})
+	if !errors.Is(err, errCategoryNotSet) {
+		t.Fatalf("expected %v, got %v", errCategoryNotSet, err)
+	}
+	_, err = b.AmendOrder(context.Background(), &AmendOrderParams{
+		OrderID:  "c6f055d9-7f21-4079-913d-e6523a9cfffa",
+		Category: "mycat"})
+	if !errors.Is(err, errInvalidCategory) {
+		t.Fatalf("expected %v, got %v", errInvalidCategory, err)
+	}
+	_, err = b.AmendOrder(context.Background(), &AmendOrderParams{
+		OrderID:  "c6f055d9-7f21-4079-913d-e6523a9cfffa",
+		Category: "option"})
+	if !errors.Is(err, currency.ErrCurrencyPairEmpty) {
+		t.Fatalf("expected %v, got %v", currency.ErrCurrencyPairEmpty, err)
+	}
+	_, err = b.AmendOrder(context.Background(), &AmendOrderParams{
+		OrderID:  "c6f055d9-7f21-4079-913d-e6523a9cfffa",
+		Category: "spot", Symbol: spotTradablePair,
+		TriggerPrice:    1145,
+		OrderQuantity:   0.15,
+		Price:           1050,
+		TakeProfitPrice: 0,
+		StopLossPrice:   0})
 	if err != nil {
 		t.Error(err)
```

Comment:
> code: 170130 message: Data sent for paramter '' is not valid.

Vague enough to look into

Reply:
{"retCode":10010,"retMsg":"Unmatched IP, please check your API key's bound IP addresses.","result":{},"retExtInfo":{},"time":1701885582052}